### PR TITLE
RFC: update lamports per sol?

### PR DESCRIPTION
**Pull request by sunnygleason**
PROBLEM:

* looks like the ratio might be changing? https://github.com/solana-labs/solana-web3.js/pull/541

SOLUTION:

* I did a quick check for usage, looks like not a ton of places
* tentative PR for if/when PR 541 goes through

### CODE REVIEW

File: src/v2/components/Accounts/Detail/index.jsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const AccountDetail = ({match}: {match: Match}) => {
       label: 'Balance',
       hint: '',
       value: `${((accountInfo.lamports || 0) * LAMPORT_SOL_RATIO).toFixed(
-        8,
+        12,
```
Comment (CriesofCarrots) [inline src/v2/components/Accounts/Detail/index.jsx]: I believe 9 should be enough decimal places

File: src/v2/constants.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
+import * as solanaWeb3 from '@solana/web3.js';
+
 export const MIN_TERM_LEN = 3;
-export const LAMPORT_SOL_RATIO = 0.0000000000582;
+export const LAMPORT_SOL_RATIO = 1.0 / solanaWeb3.SOL_LAMPORTS;
```
Comment (CriesofCarrots) [inline src/v2/constants.js]: There is also a `LAMPORT_SOL_RATIO` in api/util.js
Can you please consolidate into one constant? api is probably the right level for it.
😍  pulling in solanaWeb3.SOL_LAMPORTS, though!

**Comment by sunnygleason** [comment]
@CriesofCarrots sounds good, thank you so much!

**Comment by sunnygleason** [comment]
@CriesofCarrots merging this momentarily FYI (took your advice to move calculation totally into API).

**Pull request merged**